about.html

```diff
@@ -0,0 +1,154 @@
+                <a href=""><p>info@harmonyfest.org</p></a>
+            </div>
+     </main>
+    </section>
+
+
+    <section id="about" class="about">
+        <div class="main-program">
+            <h2>Harmony Fest:Uniting Music and Community 2023</h2>
+            <div class="main-border"></div>
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Provident est odit numquam tempora autem repudiandae porro quisquam dignissimos a placeat voluptatem error maxime adipisci sed temporibus sapiente alias, nulla rem consequuntur incidunt eligendi praesentium iusto excepturi dolore! Accusamus sint quibusdam minus cupiditate sunt labore, explicabo voluptatibus esse pariatur quo fuga.</p>
+            <div class="main-progsec">
+                <img src="images/logo.png" alt="">
+            </div>
+        </div>
+    </section>
+
+    <section class="partners-container">
+        <div class="partners">
+            <p class="event-title">See the past Harmony Fest Events</p>
+            <div class="partner-border"></div>
+            <p class="event-par">Take a look at the past two Harmony Fest Events that took place in Greenfield in HeartCity</p>
+            <div class="partner-logos">
+
+                <div class="event-img">
+                    <div class="event-info">
+                    <h2>2019</h2>
+                    <p>Harmony Fest Event in IceLand</p>
+                    </div>
+                    <div class="event-color"></div>
+                    <img src="images/event1.jpg" alt="">
+                </div>
+
+                <div class="event-img">
+                    <div class="event-info">
+                        <h2>2021</h2>
+                        <p>Harmony Fest Event in Greenfield</p>
+                    </div>
+                    <div class="event-color"></div>
+                    <img src="images/event2.jpg" alt="">
+                </div>
+
+            </div>
+        </div>
+    </section>
+
+    <section class="partners-div">
+        <p>Partner</p>
+        <div class="partner-border"></div>
+        <div class="logo-partners">
+            <div class="pat-logo">
+                <img src="images/partner1.png" alt="">
+            </div>
+            <div class="pat-logo">
+                <img src="images/partner2.png" alt="">
+            </div>
+            <div class="pat-logo">
+                <img src="images/partner3.png" alt="">
+            </div>
+            <div class="pat-logo">
+                <img src="images/partner4.png" alt="">
+            </div>
+            <div class="pat-logo">
+                <img src="images/partner5.png" alt="">
+            </div>
+        </div>
+    </section>
+
+    <section class="about-end">
+        <div class="about-end-logo">
+            <img src="images/logo.png" alt="">
+        </div>
+        <div class="about-rights">
+            <p>2023 Harmony Fest Some Rights Reserved</p>
+        </div>
+    </section>
+
```

Comment:
Ás stated by the previous reviewer, kindly use a `footer`tag for the footer section of your page here for good semantics 😄